composer.json

```diff
@@ -9,10 +9,20 @@
     },
     "require-dev": {
         "brain/monkey": "^2",
+        "brainmaestro/composer-git-hooks": "^2.8",
```

Comment:
This package installs the pre-commit hooks on running `composer install` or `composer update`. Otherwise, users would probably have to manually install it, or run a bash script to install it.